tests/test_transmart_loader.py

```diff
@@ -117,6 +126,15 @@ def collection_with_relations() -> DataCollection:
     return collection
 
 
+def get_column_values(file_path: str, column_num: int) -> List[str]:
```

Comment:
Conceptually, I would expect a function that gets column values by name, so that you can get the values for the `c_fullname` column instead of fetching column `1` and check that the first value is `c_fullname`

Reply:
thanks for the suggestion, it is now changed to get columns by name